### PR TITLE
fix: default dashboard/chart cards are vector with perfect corners

### PR DESCRIPTION
**Pull request by rusackas**
### SUMMARY

The corners of the default card/dash images had pre-rendered rounded corners, that looked goofy at certain scales. They were also PNG, so they got a little blurry at times.

Now they're SVG, with square corners, cropped by their containers' border-radius, so we should be sharp all the way around!

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

image

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [x] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by suddjian — APPROVED** [review]
hell yeah

**Review by junlincc — APPROVED** [review]

**Review by nytai — APPROVED** [review]
looks like those SVG files are missing licenses, otherwise LGTM

**Comment by rusackas** [comment]
> looks like those SVG files are missing licenses, otherwise LGTM

Whoops! Too hasty. Added! Will merge when CI passes. Thanks all!

**Pull request merged**
